test/Unit/OpenTelemetry/OpenTelemetryControllerTests.cs

```diff
@@ -0,0 +1,163 @@
+﻿using Moq;
+using System;
+using Xunit;
+
+using Microsoft.Azure.Functions.PowerShellWorker.OpenTelemetry;
+using System.Linq;
+using Microsoft.Azure.Functions.PowerShellWorker.Utility;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Test.OpenTelemetry
+{
+    using PowerShell = System.Management.Automation.PowerShell;
+
+    public class OpenTelemetryControllerTests
+    {
+        private readonly Mock<ILogger> _mockLogger = new Mock<ILogger>(MockBehavior.Strict);
+        private readonly Mock<IOpenTelemetryServices> _mockOtelServices;
+
+        public OpenTelemetryControllerTests()
+        {
+            _mockOtelServices = new Mock<IOpenTelemetryServices>(MockBehavior.Strict);
+        }
+
+        [Theory]
+        [InlineData("true", true)]
+        [InlineData("false", false)]
+        [InlineData(null, false)]
+        internal void OpenTelemetryEnvironmentVariableCheckWorks(string? environmentVariableValue, bool desiredResult)
+        {
+            try
+            {
+                Environment.SetEnvironmentVariable("OTEL_FUNCTIONS_WORKER_ENABLED", environmentVariableValue);
+                
+                Assert.Equal(desiredResult, OpenTelemetryController.IsOpenTelemetryEnvironmentEnabled());
+            }
+            finally
+            {
+                Environment.SetEnvironmentVariable("OTEL_FUNCTIONS_WORKER_ENABLED", null);
+                OpenTelemetryController.ResetOpenTelemetryModuleStatus();
+            }
+        }
+
+        [Fact]
+        internal void AddStartOpenTelemetryInvocationCommand_AddsCommands()
+        {
+            try
+            {
+                PowerShell _pwsh = PowerShell.Create();
+                var _realOTelServices = new OpenTelemetryServices(_mockLogger.Object, _pwsh);
+
+                OpenTelemetryController controller = CreateMockOpenTelemetryController();
+
+                OpenTelemetryInvocationContext context = new OpenTelemetryInvocationContext(
+                    "93d73ba2-bac9-41f9-ad31-e7ab56a6d7e1",
```

Comment:
These values look like magic constants. Do you have to use these specific values? Do they have to follow any specific format? If your code does not interpret them and does not really depend on them, I suggest replacing them with "fake parent id" and "fake trace id". In case the validations below ever fail, we will aprreciate the error message clarity.

Reply:
Done